packages/react-components/lib/map/three-fiber/image-maker.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Level } from 'api-client';
+import { useLoader } from '@react-three/fiber';
+import { Texture, TextureLoader } from 'three';
+
+export interface ImageThreeProps {
+  level: Level;
+  imageUrl: string;
+}
+
+export const ReactThreeFiberImageMaker = ({ level, imageUrl }: ImageThreeProps): JSX.Element => {
+  let texture: Texture | undefined = undefined;
+  try {
+    texture = useLoader(TextureLoader, imageUrl);
```

Comment:
This is a tricky check failure (sorry I introduced this), however I'm unsure how we can be sure the `imageUrl` is accessible in the first place, these were some of the things I found, https://www.freecodecamp.org/news/how-to-validate-urls-in-javascript/.

Reply:
It is currently working properly, could you try it?